.changeset/sweet-waves-matter.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react-brand': patch
+---
+
+Update order of River children to ensure accessible markup order and position visual above content on narro viewports
```

Comment:
Small typo here:

```suggestion
Update order of River children to ensure accessible markup order and position visual above content on narrow viewports
```

Also, do you think this should be flagged as a breaking change? Might at least be worth calling out that it will flip the order of some specific Rivers, wdyt?